errai-data-binding/src/test/java/org/jboss/errai/databinding/client/test/DataBindingIntegrationTest.java

```diff
@@ -1588,4 +1589,14 @@ public void testDeclarativeBindingWithOnKeyUpForTextInput() throws Exception {
     invokeEventListeners(view.value, "keyup");
     assertEquals(view.value.getValue(), model.getValue());
   }
+
+  @Test
+  public void testTypeInNonBiandablePakageIsNotBound() {
```

Comment:
Minor thing, but there are a couple typos here. Should probably be `testTypeInNonBindablePackageIsNotBound`.